src/AdminMenus/FormSettingsAdminSubMenu.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * Class that holds class for admin sub menu - Form Settings.
+ *
+ * @package EightshiftLibs\AdminMenus
```

Comment:
```suggestion
 * @package EightshiftForms\AdminMenus
```
I can see that here is again Libs, maybe I'm completely wrong in this and the case above it.

Reply:
fixed